examples/ompls/sci.yml

```diff
@@ -30,7 +31,5 @@ graph:
           operational-carbon: 5
           embodied-carbon: 0.02
           requests: 100
-          functional-unit-duration: 1
-          functional-unit-time: minutes
-          functional-unit: requests
+          carbon: 0.0013944444444444442
```

Comment:
`carbon` was added because of new logic on sci model. Looks like sci example was just a bit outdated. Anyway not related to my changes.